whisper/README.md

```diff
@@ -39,8 +39,51 @@ To generate a 4-bit quantized model, use `-q`. For a full list of options:
 python convert.py --help
 ```
 
-By default, the conversion script will make the directory `mlx_models/tiny`
-and save the converted `weights.npz` and `config.json` there.
+By default, the conversion script will make the directory `mlx_models`
+and save the converted `weights.npz` and `config.json` there. Note that
+this does not automatically save different versions in this folder.
+However, the downloaded models are cached for future use. 
+
+Consider a scripted run (e.g. BASH, zsh) like the below example to stream-line filemaking:
+
+ Here's a Python script that acts as a wrapper for calling `convert.py` with different parameters:
+
+```import sys
+import subprocess
+
+def run_convert(config):
```

Comment:
Do you mind removing this script? I don't think it makes sense to include. Different people will have different needs with respect to how they want serialize models.